cammac60/MacRae-Photo
# Gallery comp

### What does this PR do? 

- Adds/styles the gallery component. 
- Adds one new image to `images/main` and the matching data to the data file.
- Adds onto bio. 

----------

### How to manually test: 

- `git fetch`
- `git checkout gallery-comp`
- `npm test` to ensure all tests are passing. 
- `git start` => click on the photos button => Ensure page loads and images are clickable, ensure back button routes back to splash, ensure IG button opens a new tab with IG page. 

----------

### Tickets: 

- Resolves #20 
- Resolves #21 

----------

### Notes: 

- The images have a set height and width in pixels. This needs to be changed to accommodate for other screen sizes.  Added issue #35
